tools/core_memory_remember.py

```
@@ -99,6 +99,11 @@ async def run(
             self.logger.error(f"{self.name}: 无法获取上下文信息或 memory_runtime 实例: {e}")
             return "错误：无法确定当前会话ID，记忆写入已拒绝（严格隔离模式）。"
 
+        # --- 自动追加发送者 QQ 号到 tags ---
+        sender_id = str(event.get_sender_id())
+        if sender_id:
+            tags = list(tags) + [f"uid:{sender_id}"]
```

Ruff (0.15.13)

[warning] Consider `[*list(tags), f"uid:{sender_id}"]` instead of concatenation

Replace with `[*list(tags), f"uid:{sender_id}"]`

(RUF005)